barretenberg/cpp/src/barretenberg/benchmark/pippenger_bench/main.cpp

```diff
@@ -72,7 +72,7 @@ int pippenger()
     scalar_multiplication::pippenger_runtime_state<curve::BN254> state(NUM_POINTS);
     std::chrono::steady_clock::time_point time_start = std::chrono::steady_clock::now();
     g1::element result = scalar_multiplication::pippenger_unsafe<curve::BN254>(
-        &scalars[0], reference_string->get_monomial_points(), NUM_POINTS, state);
+        { &scalars[0], NUM_POINTS }, reference_string->get_monomial_points(), NUM_POINTS, state);
```

Comment:
does this syntax construct a span from the localation of scalars[0] up to NUM_POINTS? I assume so. It'd be good if you could document this in whatever place you see best

Reply:
commented